org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend

```diff
@@ -134,7 +143,7 @@ import org.eclipse.xtext.validation.Issue
 		]
 
 		requestManager.runWrite [ cancelIndicator |
-			val rootURI = URI.createFileURI(params.rootPath).toPath.toUri
+			val rootURI = URI.createURI(rootPath.toUri.toString)
```

Comment:
`rootURI` should be `'file://' + rootPath` where `rootPart` is an absolute path, if it so then looks good to me